src/armorblox/api/incidents_api.py

```diff
@@ -20,14 +20,15 @@ def __init__(self, config, incident_type=None):
         super().__init__(config)
         self._incident_type = incident_type
 
-    def list(self, page_token=None):
+    def list(self, page_token = None, params = None):
```

Comment:
The spaces around `=` should not be used in keyword arguments. See https://peps.python.org/pep-0008/#other-recommendations